io.github.twisterarmy.twister.json

```diff
@@ -0,0 +1,95 @@
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "twister",
+    "finish-args": [
+        "--share=network",
+        "--persist=.twister"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/*.a",
+        "/lib/*.la",
+        "/lib/cmake",
+        "/lib/pkgconfig",
+        "/share/cmake",
+        "/share/man"
+    ],
+    "modules": [
+        {
+            "name": "boost",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./bootstrap.sh --prefix=/app",
+                "./b2 install -j $FLATPAK_BUILDER_N_JOBS"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://archives.boost.io/release/1.83.0/source/boost_1_83_0.tar.gz",
+                    "sha256": "c0685b68dd44cc46574cce86c4e17c0f611b15e195be9848dfd0769a0a207628"
+                }
+            ]
+        },
+        {
+            "name": "libdb",
+            "buildsystem": "simple",
+            "build-commands": [
+                "cd build_unix && ../dist/configure --prefix=/app --enable-cxx && make -j $FLATPAK_BUILDER_N_JOBS && make install"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.oracle.com/berkeley-db/db-4.8.30.NC.tar.gz",
+                    "sha256": "12edc0df75bf9abd7f82f821795bcee50f42cb2e5f76a6a281b85732798364ef"
+                },
+                {
+                    "type": "patch",
+                    "path": "libdb.patch",
+                    "options": [
+                        "-p2"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "core",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./autotool.sh",
+                "./configure --prefix=/app --with-boost=/app --with-libdb=/app",
+                "make -j $FLATPAK_BUILDER_N_JOBS && make install"
+            ],
+            "post-install": [
+                "install -D share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg -t /app/share/icons/hicolor/scalable/apps",
+                "install -D share/applications/${FLATPAK_ID}.desktop -t /app/share/applications"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/twisterarmy/twister-core.git",
+                    "commit": "e94eb385b9303e6d91387d04bfdeb361809b640b"
+                }
+            ]
+        },
+        {
+            "name": "html",
+            "buildsystem": "simple",
+            "build-commands": [
+                "echo -e \"xdg-open http://127.0.0.1:28331\ntwisterd -htmldir=/app/share/html -rpcport=28331 -rpcuser=user -rpcpassword=pwd -rpcallowip=127.0.0.1\" > /app/twister",
+                "chmod +x /app/twister",
+                "ln -s /app/twister /app/bin/twister",
```

Comment:
why not installing it `/app/bin` directly instead.  that where executables belong.